shell/client/shell.html

```diff
@@ -899,6 +902,41 @@
   </div>
 </template>
 
+<template name="adminCaps">
+  {{#if powerboxOfferUrl}}
+  <div id="powerbox-offer-popup" class="popup">
+    <h3>Powerbox Offer</h3>
+    <div>
+      <input type="text" class="autoSelect" value="{{powerboxOfferUrl}}" readonly>
+    </div>
+  </div>
+  <div id="powerbox-offer-popup-closer" class="popup-closer"></div>
+  {{/if}}
+  <p><button id="offer-ipnetwork">Offer IpNetwork Capability</button>
+     <button id="offer-ipinterface">Offer IpInterface Capability</button><br>
+    <span class="details">These buttons create IpNetwork and IpInterface capabilities respectively, and will return webkeys that you can restore in other apps. Typically these apps will be "drivers" that offer more granular protocols such as IRC or SMTP.</span></p>
```

Comment:
When writing text for consumption by the user, we need to make sure we don't use a bunch of implementation jargon than only we understand.

How about: "These buttons create webkeys (special secret addresses) representing permission to directly access the network. Some apps -- especially drivers -- will ask for these permissions. To grant them, you'll need to copy/paste the webkey from here into the app's request box. (Eventually, this UI will be superseded by the Powerbox picker.)"